policies/curity-phantom-token-auth-inbound/schema.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "$id": "http://zuplo.com/schemas/policies/curity-phantom-token-auth-inbound.json",
+  "type": "object",
+  "title": "Curity Phantom Token Auth",
+  "isPreview": false,
+  "description": "Enables a custom code policy written in TypeScript. Change curity-phantom-token to the name of your module (without .ts extension)",
+  "required": ["handler"],
+  "properties": {
+    "handler": {
+      "type": "object",
+      "default": {},
+      "required": ["export", "module", "options"],
+      "properties": {
+        "export": {
+          "const": "default",
+          "description": "The name of the exported type"
+        },
+        "module": {
+          "const": "$import(./modules/curity-phantom-token)",
+          "description": "The module containing the policy"
+        },
+        "options": {
+          "type": "object",
+          "description": "Options to be passed to the custom policy",
+          "required": [],
+          "properties": {
+            "allowUnauthenticatedRequests": {
```

Comment:
Is this property handled automatically by Zuplo or should the support for it be implemented in the module code?